services/ui_backend_service/api/utils.py

```diff
@@ -25,12 +25,12 @@ def format_response(request: web.BaseRequest, db_response: DBResponse) -> (int,
     return db_response.response_code, response_object
 
 
-def format_response_list(request: web.BaseRequest, db_response: DBResponse, page: int, lastPage: int) -> (int, Dict):
+def format_response_list(request: web.BaseRequest, db_response: DBResponse, page: int) -> (int, Dict):
     query = {}
     for key in request.query:
         query[key] = request.query.get(key)
 
-    nextPage = min(page + 1, lastPage)
+    nextPage = page + 1
```

Comment:
could this have a check if the DBResponse body is empty, to instead not provide nextpage at all, or do we always want to provide a link?

Reply:
Excellent point, will update this.